docs/platforms/unreal/installation/installation-recommendations.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Installation Recommendations
+description: "Learn more about choosing the right installation for Unreal Engine."
+sidebar_order: 8000
+---
+
+## Installation Recommendations: GitHub Releases or Fab
+
+<Note>
+While Unreal Engine uses "plugins", we will refer to the Sentry support we offer within Unreal Engine as the Sentry Unreal Engine SDK.
+</Note>
+
+There are three common ways to install an SDK to use with Unreal Engine:
+1. Install from the [Epic Games Fab](https://www.fab.com/) ([formerly known as the marketplace](https://www.unrealengine.com/en-US/blog/fab-epics-new-unified-content-marketplace-launches-today))
+2. Download a pre-built SDK that you install (e.g. from a [GitHub Releases page](https://github.com/getsentry/sentry-unreal/releases))
+3. Clone and build the SDK yourself and install
```

Comment:
Nice. This will come in handy soon since we're working on improving our build so users can build our SDK using U reals build system.